livekit/sip.go

```diff
@@ -685,6 +686,29 @@ func (p *CreateSIPParticipantRequest) Validate() error {
 		// TODO: Validate display name doesn't contain invalid characters
 	}
 
+	// Validate destination if provided
+	if p.Destination != nil {
+		if err := p.Destination.Validate(); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func (d *Destination) Validate() error {
+	// Rule 1: If city is specified, country must be specified
+	if d.City != "" && d.Country == "" {
```

Comment:
```suggestion
	if d.City != "" && d.Country == "" && d.Region == "" {
```
I guess it's okay if either region or country is set.